kernel: bump 6.6 to 6.6.92

graysky2 opened the pull request:
Changelog: https://cdn.kernel.org/pub/linux/kernel/v6.x/ChangeLog-6.6.92

No patches needed a rebase with this release/only archive hash updated.

Build system: x86/64
Build-tested: flogic/xiaomi_redmi-router-ax6000-ubootmod, ramips/tplink_archer-a6-v3
Run-tested: flogic/xiaomi_redmi-router-ax6000-ubootmod, ramips/tplink_archer-a6-v3

graysky2 commented:
Related to the PR is the failure of `kernel_update.sh` to update the kernel version and hash which I did manually. Please see my [forum post](https://forum.openwrt.org/t/running-update-kernel-sh-from-maintainer-tools-is-no-longer-update-the-kernel-hash/232557/1) and comment there if you have any insights.

robimarko commented:
Can you please rebase this, so it can be merged?
x86 6.6 support was removed

graysky2 committed:
kernel: bump 6.6 to 6.6.91

Changelog: https://cdn.kernel.org/pub/linux/kernel/v6.x/ChangeLog-6.6.91

Removed upstreamed:
	generic/pending-6.6/730-net-ethernet-mtk_eth_soc-reset-all-TX-queues-on-DMA-.patch[1]

All other patches automatically rebased.

1. https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/commit/?h=v6.6.91&id=68f29bb97a0e0519156664cdc23e8b1f129e3254

Build system: x86/64
Build-tested: flogic/xiaomi_redmi-router-ax6000-ubootmod, ramips/tplink_archer-a6-v3
Run-tested: flogic/xiaomi_redmi-router-ax6000-ubootmod, ramips/tplink_archer-a6-v3

Signed-off-by: John Audia <therealgraysky@proton.me>
Link: https://github.com/openwrt/openwrt/pull/18835
Signed-off-by: Hauke Mehrtens <hauke@hauke-m.de>

rockdrilla commented:
@graysky2 may I ask you to update to 6.6.92?

graysky2 commented:
Yes, later today will have some time

On Thu, May 22, 2025 at 10:02 AM, Konstantin Demin ***@***.***(mailto:On Thu, May 22, 2025 at 10:02 AM, Konstantin Demin <<a href=)> wrote:

> rockdrilla left a comment [(openwrt/openwrt#18835)](https://github.com/openwrt/openwrt/pull/18835#issuecomment-2901348066)
>
> ***@***.***(https://github.com/graysky2) may I ask you to update to 6.6.92?
>
> —
> Reply to this email directly, [view it on GitHub](https://github.com/openwrt/openwrt/pull/18835#issuecomment-2901348066), or [unsubscribe](https://github.com/notifications/unsubscribe-auth/AAECSJUNGYVNRB5Q6ZQUTRD27XKGHAVCNFSM6AAAAAB5LXZT2SVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDSMBRGM2DQMBWGY).
> You are receiving this because you were mentioned.Message ID: ***@***.***>

graysky2 committed:
kernel: bump 6.6 to 6.6.92

Changelog: https://cdn.kernel.org/pub/linux/kernel/v6.x/ChangeLog-6.6.92

No patches needed a rebase with this release/only archive hash updated.

Build system: x86/64
Build-tested: flogic/xiaomi_redmi-router-ax6000-ubootmod, ramips/tplink_archer-a6-v3
Run-tested: flogic/xiaomi_redmi-router-ax6000-ubootmod, ramips/tplink_archer-a6-v3

Signed-off-by: John Audia <therealgraysky@proton.me>
Link: https://github.com/openwrt/openwrt/pull/18835
Signed-off-by: Hauke Mehrtens <hauke@hauke-m.de>

zxlhhyccc commented:
````
make[1] target/compile
 make[2] -C target/linux compile
    ERROR: target/linux failed to build.
make -r target/compile: build failed. Please re-run make with -j1 V=s or V=sc for a higher verbosity level to see what's going on
make: *** [/__w/openwrt/openwrt/openwrt/include/toplevel.mk:233: target/compile] Error 1
====== Make errors from logs/target/linux/compile.txt ======
  INSTALL /__w/openwrt/openwrt/openwrt/build_dir/target-aarch64-openwrt-linux-musl_musl/linux-rockchip_armv8/linux-6.6.92/user_headers/include
make[4]: Leaving directory '/__w/openwrt/openwrt/openwrt/build_dir/target-aarch64-openwrt-linux-musl_musl/linux-rockchip_armv8/linux-6.6.92'
*
* Restart config...
*
*
* ARM System Control and Management Interface Protocol
*
ARM System Control and Management Interface (SCMI) Message Protocol (ARM_SCMI_PROTOCOL) [Y/n/m/?] y
  Enable support for SCMI Raw transmission mode (ARM_SCMI_RAW_MODE_SUPPORT) [N/y/?] n
  Enable SCMI communication debug metrics tracking (ARM_SCMI_DEBUG_COUNTERS) [N/y/?] (NEW) make[7]: *** [scripts/kconfig/Makefile:77: syncconfig] Error 1
make[6]: *** [Makefile:697: syncconfig] Error 2
make[5]: *** [/__w/openwrt/openwrt/openwrt/build_dir/target-aarch64-openwrt-linux-musl_musl/linux-rockchip_armv8/linux-6.6.92/Makefile:798: include/config/auto.conf.cmd] Error 2
make[4]: *** [Makefile:[23](https://github.com/openwrt/openwrt/actions/runs/15196265746/job/42751218936#step:31:24)4: __sub-make] Error 2
make[3]: *** [Makefile:23: /__w/openwrt/openwrt/openwrt/build_dir/target-aarch64-openwrt-linux-musl_musl/linux-rockchip_armv8/linux-6.6.92/.modules] Error 2
make[2]: *** [Makefile:12: compile] Error 2
``````

graysky2 committed:
rockchip: armv8: add missing config option

6.6.92 requires the addition of CONFIG_ARM_SCMI_DEBUG_COUNTERS

Signed-off-by: John Audia <therealgraysky@proton.me>
Link: https://github.com/openwrt/openwrt/pull/18835
Signed-off-by: Hauke Mehrtens <hauke@hauke-m.de>

graysky2 committed:
stm32: add missing config option

6.6.92 requires the addition of CONFIG_ARM_SCMI_DEBUG_COUNTERS

Signed-off-by: John Audia <therealgraysky@proton.me>
Link: https://github.com/openwrt/openwrt/pull/18835
Signed-off-by: Hauke Mehrtens <hauke@hauke-m.de>

DragonBluep commented on target/linux/stm32/stm32mp1/config-6.6:
```diff
@@ -42,6 +42,7 @@ CONFIG_ARM_PATCH_PHYS_VIRT=y
 CONFIG_ARM_PSCI_CPUIDLE_DOMAIN=y
+# CONFIG_ARM_SCMI_DEBUG_COUNTERS is not set
 CONFIG_ARM_PSCI_FW=y
 CONFIG_ARM_SCMI_CPUFREQ=y
```
Please sort it alphabetically.

graysky2 replied:
Oops, corrected
